src/info/guardianproject/otr/app/im/app/AccountWizardActivity.java

```diff
@@ -238,28 +244,42 @@ public boolean onPrepareOptionsMenu(Menu menu) {
 
     private void buildAccountList ()
     {
+        int i = 0;
+        int accountProviders = 0;
         List<String> listProviders = helper.getProviderNames();
 
         mGoogleAccounts = AccountManager.get(this).getAccountsByType(GTalkOAuth2.TYPE_GOOGLE_ACCT);
 
-        mAccountList = new String[listProviders.size()+3][2]; //potentialProviders + google + create account + burner
-
-        int i = 0;
-
-        mAccountList[i][0] = getString(R.string.i_want_to_chat_using_my_google_account);
-        mAccountList[i++][1] = getString(R.string.account_google_full);
+        if (mGoogleAccounts.length > 0) {
```

Comment:
I was not able to configure gtalk as a standard XMPP account with ChatSecure. I tried a lot of options, but as you said, their setup is complicated and i'm not sure if it's even possible with the possibility offered by ChatSecure "regular XMPP". From my cyanogenmod-user point of view this is a very hard limitation.
( I can connect to gtalk with Conversation for example, even if i have no google account setup on the phone by just providing login/password. )